stl/inc/stdatomic.h

```diff
@@ -7,17 +7,19 @@
 #ifndef _STDATOMIC_H_
 #define _STDATOMIC_H_
 
-#if defined(RC_INVOKED) || defined(Q_MOC_RUN) || defined(__midl)
-// do nothing, see _STL_COMPILER_PREPROCESSOR in yvals_core.h
-#else // ^^^ non-compiler tools / C and C++ compilers vvv
+// do nothing for preprocessors, see _STL_COMPILER_PREPROCESSOR in yvals_core.h

```

Comment:
"do nothing for preprocessors" is insufficiently precise. We want to work with the C++ (and eventually C) preprocessors, what we're excluding here are something like "non-compilation code processing tools".

Reply:
(FWIW, I'd be happy with just "See _STL_COMPILER_PREPROCESSOR in yvals_core.h".)